packages/mgt-components/src/components/mgt-todo/mgt-todo.ts

```diff
@@ -171,11 +172,13 @@ export class MgtTodo extends MgtTasksBase {
     );
 
     const completedTaskTemplates = repeat(
-      completedTasks.sort((a, b) => {
-        return new Date(a.lastModifiedDateTime).getTime() - new Date(b.lastModifiedDateTime).getTime();
-      }),
+      completedTasks
+        .sort((a, b) => {
+          return new Date(a.lastModifiedDateTime).getTime() - new Date(b.lastModifiedDateTime).getTime();
```

Comment:
I get the error "The left-hand side of an arithmetic operation must be type 'any', 'number', 'bigint' or an enum type" which seems to occur because typescript doesn't consider Date objects valid left-hand or right-hand sides to arithmetic operations. Using the getTime() works by returning a number